### PR TITLE
added Unit field to form render and Yup validation schema

### LINKED ISSUE #200: FrontEnd: Add Second Address line for Unit
Add Unit to Address line in front end. 
Line should be optional 
"unit": string

### PR DESCRIPTION
**Pull request by mswanson-me**
Placeholder text in the form is set as "Unit number", but I am not sure if that should be "Apartment/Unit number" or "Apartment or Unit number" or some variation.

### CODE REVIEW

**Review by Chris-Boe — CHANGES_REQUESTED** [review]
so according to the mockup (page 15 of https://drive.google.com/file/d/1YqboQogczYm1HkyRqEtVSzeQ61T9hWU2/view) it looks like "Units" is the number of Units that a property contains. The mockup also has it as the last line.

Let's make those changes to align with the mockup. Once this page gets hooked up to the API, we will also need to have "Units" added as a parameter to the Property resource on the backend

**Comment by mswanson-me** [comment]
The requested changes have been made.

File: src/views/addProperty.js [hunk 1]
```diff
@@ -132,6 +135,19 @@ export class AddProperty extends Component {
...
+                                                onChange={handleChange}
+                                                value={values.units}
+                                                placeholder="Number of units"
+                                                error={errors.units}
+                                            />
+                                            {errors.units ? (<div className="error-message">{errors.units}</div>) : null}
+                                        </div>
                                         {/* This field needs to be included after
                                             the units parameter is added to the api endpoint */}
                                         {/* <div className="form-row columns">
```
Comment (Chris-Boe) [inline src/views/addProperty.js]: Looks like there's an older commented out version of this field right here so we can get rid of it 😄

**Review by Chris-Boe — CHANGES_REQUESTED** [review]

File: src/views/addProperty.js (hunk 1 above)
Comment (mswanson-me) [reply src/views/addProperty.js]: LOL that's pretty funny. Fixing this now.

Comment (mswanson-me) [reply src/views/addProperty.js]: Give it one more shot.

**Review by Chris-Boe — APPROVED** [review]
Nice job!

**Pull request merged**